### PR TITLE
Use stage WS URL when on console.dev

### PR DESCRIPTION
**Pull request by karelhala**

### CODE REVIEW

File: frontend/src/sdk/useAppConfiguration.ts [hunk 1]
```diff
@@ -8,6 +8,20 @@ import { useChrome } from '@redhat-cloud-services/frontend-components/useChrome'
 
 type AppConfigurations = React.ComponentProps<typeof AppInitSDK>['configurations'];
 
+// TODO: remove once RHCLOUD-24173 is fixed and return just `wss://${location.host}${prefix || '/wss/k8s'}`
+const getWsURL = ({ wsPrefix, pathPrefix }: { wsPrefix?: string; pathPrefix?: string } = {}) => {
```
Comment (florkbr) [inline frontend/src/sdk/useAppConfiguration.ts:12]: 👍 for moving this to a function

File: frontend/config/dev.webpack.config.js [hunk 2]
```diff
@@ -65,7 +65,9 @@ const webpackProxy = {
   customProxy: [
     {
       context: (path) => path.includes('/api/k8s/registration'),
-      target: 'https://registration-service-toolchain-host-operator.apps.appstudio-stage.x99m.p1.openshiftapps.com',
+      target: process.env.STAGE
+        ? 'https://registration-service-toolchain-host-operator.apps.stone-stg-host1.hjvn.p1.openshiftapps.com/'
```
Comment (florkbr) [inline frontend/config/dev.webpack.config.js:69]: @karelhala does the trailing slash matter here?

Comment (karelhala) [reply frontend/config/dev.webpack.config.js]: Good point!

**Comment by florkbr** [comment]
@karelhala linter is angry :(

```
[0m[4m/go/src/github.com/openshift/hac-core/frontend/src/Navigation.ts[24m[0m
[0m  [2m6:8[22m  [33mwarning[39m  Using exported name 'useChrome' as identifier for default export  [2mimport/no-named-as-default[22m[0m
[0m[0m
[0m[4m/go/src/github.com/openshift/hac-core/frontend/src/sdk/useActivePlugins.ts[24m[0m
[0m  [2m2:8[22m  [33mwarning[39m  Using exported name 'useChrome' as identifier for default export  [2mimport/no-named-as-default[22m[0m
[0m[0m
[0m[4m/go/src/github.com/openshift/hac-core/frontend/src/sdk/useAppConfiguration.ts[24m[0m
[0m  [2m15:59[22m   [31merror[39m  Insert `;`  [2mprettier/prettier[22m[0m
[0m  [2m19:103[22m  [31merror[39m  Insert `;`  [2mprettier/prettier[22m[0m
[0m  [2m23:2[22m    [31merror[39m  Insert `;`  [2mprettier/prettier[22m[0m
[0m[0m
[0m[4m/go/src/github.com/openshift/hac-core/frontend/src/sdk/usePluginStore.ts[24m[0m
[0m  [2m2:8[22m  [33mwarning[39m  Using exported name 'useChrome' as identifier for default export  [2mimport/no-named-as-default[22m[0m
[0m[0m
[0m[31m[1m✖ 8 problems (5 errors, 3 warnings)[22m[39m[0m
[0m[31m[1m[22m[39m[31m[1m  5 errors and 0 warnings potentially fixable with the `--fix` option.[22m[39m[0m
[0m[31m[1m[22m[39m[0m
error Command failed with exit code 1.
```

**Comment by florkbr** [comment]
This seems fine to me. We will need to follow back up on removing all these hard coded URLs. I'm approving to unblock the team.

**Pull request merged**